sdk/digitaltwins/Azure.DigitalTwins.Core/src/Queries/QueryBuilders/FromQuery.cs

```diff
@@ -37,7 +37,7 @@ public WhereStatement From(AdtCollection collection)
         /// </summary>
         /// <param name="collection"> The name of the collection. </param>
         /// <returns> ADT query with select and from clause. </returns>
-        public WhereStatement From(string collection)
```

Comment:
I think either one of these is fine, I like `FromCustom` and I liked the string overload as well.